src/gentropy/assets/schemas/study_index.json

```diff
@@ -35,6 +35,16 @@
       "nullable": true,
       "metadata": {}
     },
+    {
+      "name": "diseaseIds",
```

Comment:
I hope nobody misinterprets this field. I'd love the day when we find a way to document the schemas

Reply:
Do you mean diseaseFromSource vs. mappedDisease vs disease? Haha, that is indeed quite confusing.

Reply:
indeed